frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -2328,10 +2328,10 @@ private void prepareProjectDuplicate(RenderRequest request) {
                 String department = user.getDepartment();
                 Project newProject = PortletUtils.cloneProject(emailFromRequest, department, client.getProjectById(id, user));
                 setDefaultRequestAttributes(request, newProject.getBusinessUnit());
-                Map<String, String> sortedAdditionalData = getSortedMap(newProject.getAdditionalData(), true);
-                newProject.setAdditionalData(sortedAdditionalData);
                 setAttachmentsInRequest(request, newProject);
                 PortletUtils.setCustomFieldsEdit(request, user, newProject);
+                newProject.unsetModifiedBy();
+                newProject.unsetModifiedOn();
```

Comment:
This isn't the right place, Please unset all the attributes in `cloneRelease` method of  `PortletUtils` class.

![image](https://user-images.githubusercontent.com/50870174/206685689-80401a63-6815-4732-99e9-850ad41dc1d7.png)

Reply:
done.